### DISCUSSION TITLE
How to autofill credit/debit-card details in inputfield in WebView in RN iOS app?

### DISCUSSION THREAD

**Discussion by ThushalIntervest** [react-native-webview/react-native-webview#2990] [General]
I'm developing React native app in which there is webview there will be Payment View. I have credit/debit card details like card number, exp date, etc. I want to populate credit/debit card details into the payment page input fields.

Android is working fine with the GPAY.
The default Safari browser on iPhone is able to autofill credit/debit cards if saved.

But inside the iOS APP web view payment page auto-filling in card details is not working. 
Can I know why is that?

**Comment by temelkovski** [comment]
> I'm developing React native app in which there is webview there will be Payment View. I have credit/debit card details like card number, exp date, etc. I want to populate credit/debit card details into the payment page input fields.
> 
> Android is working fine with the GPAY. The default Safari browser on iPhone is able to autofill credit/debit cards if saved.
> 
> But inside the iOS APP web view payment page auto-filling in card details is not working. Can I know why is that?

I'm having the same issue, have you found a solution?

**Comment by gus-delpino** [comment]
Did anybody find a solution to this?

**Comment by mocon** [comment]
Same issue here.

**Comment by thomasdierickx** [comment]
I am encountering the same issue. Google Pay is giving my saved cards as an option to use, but on my iPhone, nothing is being presented, even when I go through a transaction, the details are not saved as well. So every time this will need to be filled in again.
